src/cloud/docker/docker-mode-developer.md

```diff
@@ -65,7 +55,7 @@ To launch the Docker environment in developer mode:
 
 1. _Optional_: Configure the Docker global variables in the `docker-compose.yml` file. For example, you can [enable and configure Xdebug][xdebug].
```

Comment:
For disabling and enabling php-extensions need to use .magento.app.yaml
https://devdocs.magento.com/cloud/docker/docker-development-debug.html

Reply:
The xdebug instructions actually show how to enable that extension, and pass in the proper variables.  This test quoted is pointing to that example. XDebug enablement, and other php extensions, may require passing in configuration, the xdebug instructions show this well.